consensus/state.go

```diff
@@ -1603,7 +1603,7 @@ func (cs *State) enterCommit(height int64, commitRound int32) {
 	// If we don't have the block being committed, set up to get it.
 	if !cs.ProposalBlock.HashesTo(blockID.Hash) {
 		if !cs.ProposalBlockParts.HasHeader(blockID.PartSetHeader) {
-			logger.Info(
+			logger.Debug(
```

Comment:
This looks to me an unlikely situation. I would preserve info, even warning if we have that level.